From 8945502c1ab7584cd914716f1bfc0178ca5d74a1 Mon Sep 17 00:00:00 2001
From: Alex Cruz <mazdarider23@gmail.com>
Date: Tue, 24 Jan 2017 11:14:46 +0100
Subject: [PATCH 33/54] Expose external qs tile tint color

This should allow themers to get around issues like this (see pic below)

https://i.imgur.com/cG2OzRT.jpg

Change-Id: If6dbf9ab29f8007d85a3c45524b1cf4ba1b032fb
---
 packages/SystemUI/res/values/projekt_colors.xml                         | 2 ++
 .../SystemUI/src/com/android/systemui/qs/customize/TileQueryHelper.java | 2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/packages/SystemUI/res/values/projekt_colors.xml b/packages/SystemUI/res/values/projekt_colors.xml
index 2e8308a26b1..9bd5fed5498 100644
--- a/packages/SystemUI/res/values/projekt_colors.xml
+++ b/packages/SystemUI/res/values/projekt_colors.xml
@@ -30,4 +30,6 @@
     <color name="qs_battery_graph_text_color">#66FFFFFF</color>
     <!-- QS battery accent color -->
     <color name="qs_battery_accent">@*android:color/accent_device_default_light</color>
+    <!-- External QS tile tint color -->
+    <color name="external_qs_tile_tint_color">@android:color/white</color>
 </resources>
diff --git a/packages/SystemUI/src/com/android/systemui/qs/customize/TileQueryHelper.java b/packages/SystemUI/src/com/android/systemui/qs/customize/TileQueryHelper.java
index 0cd6490614b..6fabc6101ca 100644
--- a/packages/SystemUI/src/com/android/systemui/qs/customize/TileQueryHelper.java
+++ b/packages/SystemUI/src/com/android/systemui/qs/customize/TileQueryHelper.java
@@ -177,7 +177,7 @@ public class TileQueryHelper {
                     continue;
                 }
                 icon.mutate();
-                icon.setTint(mContext.getColor(android.R.color.white));
+                icon.setTint(mContext.getColor(R.color.external_qs_tile_tint_color));
                 CharSequence label = info.serviceInfo.loadLabel(pm);
                 addTile(spec, icon, label != null ? label.toString() : "null", appLabel, mContext);
             }
-- 
2.11.0

